config/initializers/sidekiq_alive.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+if Rails.env.production?
```

Comment:
```suggestion
return unless Rails.env.production?
```
I think it will be easier to read with a guard clause